docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -86,6 +86,55 @@ to discuss reasonable assumptions with subject matter experts to determine the
 most appropriate measure to which to apply the GBD risk effects in our vivarium
 simulations.
 
+Starting in GBD 2021, some continuous risk exposures were modeled with general
+relationships between the exposure level and the relative risk (going
+beyond the log-linear relationship assumed for previous iterations).
+Interpreting the GBD estimates is straightforward, once you have
+chased down all of the necessary definitions.  The relevant estimates
+include a column for exposure level, as well as columns for 500 draws
+of relative risk values at each exposure level.  The GBD 2021 PAF
+calculator often selected a TMREL for each draw from a uniform
+distribution, but for some risk factors, analysts provided draws for
+the TMREL as well.  The precise calculation to go from exposure levels
+and GBD-recorded risks to a function suitable for use as
+:math:`f_{rr}` as defined below are perhaps most clearly represented
+as python code:
```

Comment:
Can we add a link to the Central Comp code that you looked at to figure out what they did?